packages/react-storage/src/components/StorageBrowser/actions/handlers/utils.ts

```diff
@@ -1,12 +1,13 @@
 import { TransferProgressEvent } from 'aws-amplify/storage';
+import { LocationAccess as AccessGrantLocation } from '../../storage-internal';
```

Comment:
I think we should change this from JS and bringing it in. Separate PR followup i guess